rllib/env/multi_agent_env.py

```diff
@@ -389,7 +389,7 @@ def __init__(self, config: EnvContext = None):
             #  with data fields.
             if config is None:
                 config = {}
-            num = config.pop("num_agents", 1)
+            num = config.get("num_agents", 1)
```

Comment:
Overrides the `EnvSpec` for the next environment in vectorized environments.

Reply:
Good catch. I used `pop` here to make sure the underlying user's Env doesn't get confused due to the extra c'tor arg. But yeah, doesn't matter so much, this is only an internal dev API anyways.